OpenRA.Game/Map/Map.cs

```diff
@@ -265,7 +265,7 @@ public static string ComputeUID(IReadOnlyPackage package)
 			try
 			{
 				foreach (var filename in contents)
-					if (filename.EndsWith(".yaml") || filename.EndsWith(".bin") || filename.EndsWith(".lua"))
+					if (filename.EndsWith(".yaml") || filename.EndsWith(".bin") || filename.EndsWith(".lua") || filename == "map.png")
```

Comment:
Noting the discussion from discord so that this PR won't be merged before a fix: this will break RC map queries unless we gate it on a map version upgrade (keeping format 11 maps using the existing UID calculation).

Reply:
Separated this change into its own commit

Reply:
For the purposes of this PR about dynamic map updates, shall we remove the change in `ComputeUID` for now so we can deal with that separately?

Reply:
sure I'll remove it. It is dealt with by #19891